panel.go

```diff
@@ -313,16 +315,22 @@ type (
 		TextType string `json:"text"`
 		Value    string `json:"value"`
 	}
+	ColumnStyleValueMap struct {
```

Comment:
There is also `op`: https://github.com/grafana/grafana/blob/844ec82eb0dfac3fba92106ddfeb38a72286fca3/devenv/dev-dashboards/datasource-testdata/demo1.json#L321